index.js

```diff
@@ -1,26 +1,152 @@
 // 1.Fibonacci
 
-// function fibo(n) {
-//   return n <= 1 ? n : fibo(n - 1) + fibo(n - 2);
-// }
-// fibo(5);
-// fibo(6);
-// fibo(7);
+function fibo(n) {
+  return n <= 1 ? n : fibo(n - 1) + fibo(n - 2);
+}
+fibo(5);
+fibo(6);
+fibo(7);
 
 // 2. Implementing with reduce
 
 //concat;
+const arr = [1, 2];
+const arr1 = [3, 4];
+const arr2 = 5;
+
+const concatFn = (...array) =>
```

Comment:
instead of  ...array, should be (array, ...params), but let's keep it simple .. function accepts just two arguments (array, willBeConcatenated)

willBeConcatenated - can be arryay , or something else .. 

there is easier method to identify if we have an array .. Array.isArray(array)